Source/Painting/SvgPatternServer.cs

```diff
@@ -163,71 +157,84 @@ public override Brush GetBrush(SvgVisualElement renderingElement, ISvgRenderer r
                 curr = SvgDeferredPaintServer.TryGet<SvgPatternServer>(curr.InheritGradient, renderingElement);
             } while (curr != null);
 
-            var childElem = chain.Where(p => p.Children != null && p.Children.Count > 0).FirstOrDefault();
-            if (childElem == null) return null;
-            var widthElem = chain.Where(p => p.Width != null && p.Width != SvgUnit.None).FirstOrDefault();
-            var heightElem = chain.Where(p => p.Height != null && p.Height != SvgUnit.None).FirstOrDefault();
-            if (widthElem == null && heightElem == null) return null;
+            var hasChildren = chain.Where(p => p.Children.Count > 0).FirstOrDefault();

```

Comment:
`hasChildren` suggests a boolean, better use something like `firstChild`, or change the expression to get a boolean, e.g. 
```python
var hasChildren = chain.Where(p => p.Children.Count > 0).FirstOrDefault() != null;
```

Reply:
@mrbean-bremen 

Thank you for your review.
Variable names were not decided by me, so I will get your idea !!